frontend/src/common/enum/idp.enum.tsx

```diff
@@ -0,0 +1,5 @@
+export enum IDP {
```

Comment:
I prefer writing enums in typescript this way:

export const IDPS = {
    BCEID: "bceidboth",
    IDIR: "idir",
} as const;

export type IDP = typeof IDPS[keyof typeof IDPS];

Also, we probably need to replace all instances of "bceidboth" and "idir" literal string values throughout the frontend with IDPS.BCEID and IDPS.IDIR respectively

Reply:
It would also be better if you can create and put this under "frontend/src/common/types/idp.ts" instead of the current file and path.

Reply:
neat idea - that looks like a better spot. Updated.